site/content/entry/yuzu-progress-report-jun-2020/index.md

```diff
@@ -0,0 +1,174 @@
+
+By [re-enabling Compute Shaders](https://github.com/yuzu-emu/yuzu/pull/4025) and [changing the way yuzu handles Geometry Shaders](https://github.com/yuzu-emu/yuzu/pull/4031), [Morph](https://github.com/Morph1984) closed a big box of issues that previously affected only our Intel GPU users.
+
+
+{{< single-title-imgs
+    "Gone are the rainbows on Intel (Octopath Traveler)"
+    "./octopath-before.png"
+    "./octopath-fixed.png"
+  >}}
+  
+&nbsp;
+   
+{{< single-title-imgs
+    "No more T-posing (Super Smash Bros Ultimate)"
+    "./ssbu-tpose.png"
+    "./ssbu-fixed.png"
+  >}}
+
+[Volca](https://github.com/VolcaEM) [stubbed an input instruction](https://github.com/yuzu-emu/yuzu/pull/4032) allowing `Minecraft: Nintendo Switch Edition` to go in-game.
+
+{{< imgs
+    "./minecraft.png| Don't dig too deep! (Minecraft: Nintendo Switch Edition)"
+  >}}
+  
+Not everything has to be 3D games. [Morph](https://github.com/Morph1984) [replaced the default texture filtering behaviour](https://github.com/yuzu-emu/yuzu/pull/4081) to achieve crisp graphics on 2D games like `Undertale`.
+
+{{< single-title-imgs
+    "Crisp, just the way I like it"
+    "./undertale-before.png"
+    "./undertale-after.png"
+  >}}
+  
+Last on our bug fixes section. Old AMD GPUs from 2012 lack the OpenGL `GL_EXT_texture_shadow_lod` extension, leading to shaders that failed to build in `The Legend of Zelda: Breath of the Wild`. As described [here](https://github.com/KhronosGroup/SPIRV-Cross/issues/207), by using the [textureGrad function](https://github.com/yuzu-emu/yuzu/pull/4129), [Morph](https://github.com/Morph1984) achieved similar results, improving compatibility with such old devices.
+
+## Vulkan improvements
+
+`OCTOPATH TRAVELER` got some Vulkan love thanks to [Rodrigo](https://github.com/ReinUsesLisp). By implementing a [missing image format](https://github.com/yuzu-emu/yuzu/pull/4033) and [storage texels and atomic image operations](https://github.com/yuzu-emu/yuzu/pull/4034) the game can now reach in-game status in Vulkan, and with some very beautiful visuals!
+
+{{< imgs
+    "./octopath.png| This is how nostalgia makes us remember the SNES era (OCTOPATH TRAVELER)"
+  >}}
+
+[Rodrigo](https://github.com/ReinUsesLisp) also worked on [multiple](https://github.com/yuzu-emu/yuzu/pull/4110) [performance](https://github.com/yuzu-emu/yuzu/pull/4111) optimizations for the Vulkan API. This results in lower frametimes and higher framerates on average.
+
+## User Interface changes
+
+An old misconception our users have with yuzu was the `Internal Resolution` combobox in the graphics settings. This was carried over from [Citra's](https://citra-emu.org/) UI, but it wasn't coded to do anything in yuzu. So, lots of users set it up at higher values expecting an image quality increase that never happened. [Morph](https://github.com/Morph1984) decided it was time to [remove it for good](https://github.com/yuzu-emu/yuzu/pull/3966), and so he did.
```

Comment:
QComboBox is what Qt calls it, but I'm not sure that is an English word
Also, it was used in the september patreon release, so I'd add a "mention" about that here
```suggestion
An old misconception our users had with yuzu had to do with the `Internal Resolution` setting in the graphics settings. This was carried over from [Citra's](https://citra-emu.org/) UI, but it wasn't coded to do anything in yuzu. So, lots of users set it up at higher values expecting an image quality increase that could never happen. [Morph](https://github.com/Morph1984) decided it was time to [remove it for good](https://github.com/yuzu-emu/yuzu/pull/3966), and so he did.
```

Reply:
The more we mention the scaler, the more they insist on it, and Rodrigo is on more important stuff that that for now.